src/randoop/main/GenInputsAbstract.java

```diff
@@ -409,17 +373,12 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean dont_output_tests = false;
   
   /**
-   * Output sequences even if they do not complete execution.
-   * 
-   *  Randoop's default behavior is to output only tests consisting of
-   *  method call sequences that execute every statement, rather than throwing
-   *  an exception or failing a contract check before the last statement.
+   * Indicate which classes that any generated test must include. Tests that
+   * do not have classes matching the regular expression are not written to 
+   * the test suites. 
    */
-  @Option("Output sequences even if they do not complete execution")
-  public static boolean output_nonexec = false;
-  
-  @Option("specifies regex of classes that must be in any regression tests")
-  public static Pattern test_classes = null;
+  @Option("Regular expression for class names that must occur in generated tests")
```

Comment:
Yes, it only affects what goes into the output lists.

I am a little concerned that it is filtering tests that don't have any instances of a class --- since the only way that a variable would match a class name is if the value is an instance with that type. So, I think we wouldn't get any tests of purely static methods or fields.